.prettierrc

```diff
@@ -0,0 +1,15 @@
+{
+	"useTabs": true,
+	"tabWidth": 4,
+	"arrowParens": "avoid",
+	"trailingComma": "none",
```

Comment:
If we want to match [config from eslint/rewrite](https://github.com/eslint/rewrite/blob/main/prettier.config.js), we should remove `"trailingComma": "none"`.

Reply:
hmm sure, however this will require needing to disabled the eslint `comma-dangle` rule, which doesn't like hanging commas. please advise if i can proceed.

Reply:
it's not as per the `npx eslint --print-config foo.js` output i mentioned in above comment.